src/Blueprint/Blueprint/Blueprint.format.ps1xml

```diff
@@ -183,6 +183,22 @@
                 <Label>LockMode</Label>
                   <ScriptBlock>$_.Locks.Mode.ToString()</ScriptBlock>
               </ListItem>
+				<ListItem>
```

Comment:
Can you fix indention?

Reply:
+1

---
In reply to: [413182628](https://github.com/Azure/azure-powershell/pull/11661#discussion_r413182628) [](ancestors = 413182628)

Reply:
Not sure why it shows up that way. Looked fine on my IDE. Will check and fix it.

Reply:
It was tabs vs spaces btw.

---
In reply to: [413225736](https://github.com/Azure/azure-powershell/pull/11661#discussion_r413225736) [](ancestors = 413225736)

Reply:
@majastrz Yep, took me a while to realize that.